dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/strains/ClusterDrawable.java

```diff
@@ -37,14 +67,35 @@
 	 */
 	public ClusterDrawable(ColorServer colorServer, Cluster cluster) {
 		this.cluster = cluster;
+		this.properties = new LinkedHashMap<>();
 		this.sources = cluster.getNodes().stream()
 				.flatMap(e -> e.getSources().stream())
 				.collect(Collectors.toSet());
-
 		label = new Text(Integer.toString(cluster.getNodes().size()));
+		initProperties();
+		setOnMouseClicked(e -> AbstractView.setLastClicked(this));
 		draw(colorServer);
 	}
 
+	private void initProperties() {
+		properties.put(ClusterPropertyTypes.TITLE, null);
+		List<String> nodeids = cluster.getNodes().stream()
```

Comment:
I know you hate naming comments :grimacing: but still I'd like this to be renamed to `nodeIds`.